Makefile

```diff
@@ -15,6 +15,7 @@ GIT_VERSION     := $(shell git describe --dirty --tags --always --match='v*')
 VERSION         ?= $(GIT_VERSION)
 LDFLAGS         := "-w -s -X 'github.com/sustainable-computing-io/kepler/pkg/version.Version=$(VERSION)'"
 ROOTLESS	?= false
+KPROBE_SYM_NAME = $(shell grep finish_task_switch /proc/kallsyms | awk -F' ' '{print $$3}')
```

Comment:
the build system and runtime system are not always the same, building from a static name won't be portable when the target system uses different name

Reply:
i was thinking since we are building it from the multistage Dockerfile, we could control the images to have the same kernel.

the current method of checking one function name after another is also at build time, and at run time the symbol could be different. we should care about the symbols in kernel running on the host machine. 
how about we find the exact symbol name at runtime, and then attach kprobe at that symbol?

Reply:
Personally I think this level of dependency belongs to ebpf library. Requiring /proc/kallsyms brings more dependencies on the kernel.

Reply:
+1 , I think we should avoid this kind of dependency if we can use code to workaround multple options

Reply:
the multiple options we look for are static names, and may be different from target system also.